README.md

```diff
@@ -224,6 +224,19 @@ You can analyze the effect of Copilot in different languages ​​and different
 
 # Deployment
 
+## Azure DevOps
+If you are using Azure DevOps, make sure you change the name of the service connection to the name of your service connection.  You will need to change line 
+30 and 45 of the azure-dev.yml file.
+
+To create a service connection you can use the azd pipeline config --provider azdo command from the terminal.  You can read more here:
+https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/pipeline-azure-pipelines.
+
+You will need to install the "Install azd" extension from the [marketplace](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.azd) in your Azure DevOps organization if you haven't already done so.
+
+If you don't use the azd command you will need to create the DevOps variables yourself.  
```

Comment:
I would get rid of this line and tell them they have to make all the variables manually